charts/dragonfly/values.yaml

```diff
@@ -941,10 +941,14 @@ manager:
   ingress:
     # -- Enable ingress
     enable: false
+    # -- Ingress class name
+    className: ""
```

Comment:
`className` and `pathType` need k8s 1.18+

Reply:
Note added to docs

Reply:
Detecting if we can add className is already implemented  [here](https://github.com/dragonflyoss/helm-charts/pull/71/files#diff-dff1127d01114f9648a56cc0d44794d52b7bf497c2786c33942cc13c2bb9f94cR23), pathType [here](https://github.com/dragonflyoss/helm-charts/pull/71/files#diff-dff1127d01114f9648a56cc0d44794d52b7bf497c2786c33942cc13c2bb9f94cR42) We only need to check if w have k8s >=1.18

Reply:
Could you elaborate what do you expect to be done here in broad spectrum?